src/DiscordBot.py

```diff
@@ -206,13 +207,42 @@ async def on_ready():
     """
     logger.info(f"Logged in as {bot.user.name}")
 
-    github_internship = GitHubUtilities(
-        token=GITHUB_TOKEN, repo_name="SimplifyJobs/Summer2025-Internships", isSummer=True, isCoop=True
+    try:
+        latest_repo = get_latest_internship_repo()
+        logger.info(f"Using latest internship repository: {latest_repo}")
+
+        github_internship = GitHubUtilities(
+            token=GITHUB_TOKEN, repo_name=f"SimplifyJobs/{latest_repo}", isSummer=True, isCoop=True
+        )
+        github_newgrad = GitHubUtilities(token=GITHUB_TOKEN, repo_name="SimplifyJobs/New-Grad-Positions")
+        job_utilities = JobsUtilities()
+        scheduled_task.start(github_internship, github_newgrad, job_utilities)  # Start the loop
+        health_check_task.start()  # Start health check task
+    except Exception as e:
+        logger.error(f"Failed to initialize bot with latest repository: {e}", exc_info=True)
+
+def get_latest_internship_repo():
```

Comment:
This method should not exist exist within main.py. It should exist within JobsUtilities.py

Reply:
I agree 👍